jbot-example/src/test/java/example/jbot/slack/SlackBotTest.java

```diff
@@ -47,7 +47,7 @@ public class SlackBotTest {
     private TestBot bot;
 
     @Rule
-    public OutputCapture capture = new OutputCapture();
+    public OutputCaptureRule capture = new OutputCaptureRule();
```

Comment:
Was this change done because Spring's latest version deprecated `OutputCapture` and changed it to `OutputCaptureRule`?

Reply:
Exactly right.  The old OutputCapture got hidden and was not possible to use anymore.